Tests/MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -36,6 +36,7 @@ class MapboxCoreNavigationTests: XCTestCase {
     
     func testNavigationNotificationsInfoDict() {
         navigation = MapboxNavigationService(route: route, routeIndex: 0, routeOptions: routeOptions, directions: directions, simulating: .never)
+        try? Navigator.shared.navigator.resetRideSession()
```

Comment:
Would it work to reset the session inside `tearDown()` instead of repeating it for each test?

Reply:
Yes. I tried to do so. But because the `Navigator.shared.navigator` should be called every time after we set credentials of `Navigator`, through the `MapboxNavigationService`, or `PassiveLocationDataSource` initialization. But inside each test case, the initialization of `MapboxNavigationService` maybe different. So it would be hard to put in the `tearDown()`

Reply:
Yes, sure. Thanks for the suggestion. I'll work on it.